source/includes/_186-authorization-configurations.md.erb

```diff
@@ -0,0 +1,5 @@
+# Authorization Configuration
+
+The Authorization configuration API allows admin users to manage authorization configuration, The authorization configuration will later be used by a plugin to authorize a user in the GoCD server.
```

Comment:
That should be a full stop, not comma.

Change `, The authorization configuration will later...` to `. , This config will later...`